wgpu/_classes.py

```diff
@@ -206,6 +206,13 @@ def wgsl_language_features(self) -> set:
         # Looks like at the time of writing there are no definitions for extensions yet
         return set()
 
+    @apidiff.add("Provide a low-level way to let wgpu render to screen")
+    def get_canvas_context(self, present_info: dict) -> GPUCanvasContext:
+        """Get the GPUCanvasContext object for the appropriate backend."""
+        from .backends.auto import gpu
+
+        return gpu.get_canvas_context(present_info)
```

Comment:
Can you explain what this is for? It's not apparent. Most likely my mental model of the wgpu object model is not accurate...